tensorboard/backend/event_processing/event_file_loader.py

```diff
@@ -49,8 +51,12 @@ def Load(self):
     tf.logging.debug('Loading events from %s', self._file_path)
     while True:
       try:
-        with tf.errors.raise_exception_on_not_ok_status() as status:
-          self._reader.GetNext(status)
+        if len(inspect.getargspec(self._reader.GetNext).args) is 0:
```

Comment:
Does checking for length 0 work?  When i was looking at this, "self" is considered the first argument (even though it's a bound function):

```python
>>> inspect.getargspec(reader.GetNext).args
['self', 'status']
```

Oops, went back to look at my example and I got the negation wrong, my bad.  I should have said "not args[1:]" but at that point an explicit length check is probably clearer.